app/bundles/EmailBundle/RabbitMQ/Pull.php

```diff
@@ -109,9 +109,11 @@ public function init($queue = false, $general_connect = null, $use_ack = null, $
 				$ctimeout = $params['ctimeout'] ?? $this->factory->getParameter('mailer_rabbitmq_timeout_connect', 3);
 				$ptimeout = $params['ptimeout'] ?? $this->factory->getParameter('mailer_rabbitmq_timeout_pull', self::$default_read_timeout_seconds);
 				$heartbeat = $params['heartbeat'] ?? $this->factory->getParameter('mailer_rabbitmq_heartbeat_pull', self::$default_enable_heartbeat_seconds);
+				$use_ssl = $this->factory->getParameter('mailer_rabbitmq_use_ssl', false);
+                $ssl_options = ['verify_peer' => $this->factory->getParameter('mailer_rabbitmq_ssl_options_verify_peer', true), 'verify_peer_name' =>  $this->factory->getParameter('mailer_rabbitmq_ssl_options_verify_peer_name', true), 'allow_self_signed' => $this->factory->getParameter('mailer_rabbitmq_ssl_options_allow_self_signed', false)];
```

Comment:
Is there a way for the default parameters here being read from app/bundles/EmailBundle/Config/config.php ? Because changing them there will now not have an effect, which will be confusing to debug.

Reply:
Or does getParameter do exactly that?

Reply:
@floening  yes, getParameter is getting config values, and using the default (second argument) if the key is missing in config.